swap/src/database/sqlite.rs

```diff
@@ -303,6 +303,56 @@ impl Database for SqliteDatabase {
         result
     }
 
+    async fn insert_buffered_transfer_proof(
+        &self,
+        swap_id: Uuid,
+        proof: TransferProof,
+    ) -> Result<()> {
+        let mut conn = self.pool.acquire().await?;
+        let swap_id = swap_id.to_string();
+        let proof = serde_json::to_string(&proof)?;
+
+        sqlx::query!(
+            r#"
+            insert into buffered_transfer_proofs (
```

Comment:
Nitpick: I feel like sql keywords should be capitalized

Reply:
Fair point